### DISCUSSION TITLE
Extracting text from libreoffice PDFs

### DISCUSSION THREAD

**Discussion by shueffner** [pymupdf/PyMuPDF#810] [Looking for help] [answered]
Thank you and the community for all the hard work, and the great documentation and examples!

I'd like to use pymupdf to extract text and images from PDFs created by Docsplit (so, libreoffice) by converting MS Office documents to PDF.
Is there anything I need to be aware of? Does libreoffice generally generate "nice", well-formed PDFs? Would the extracted text be naturally in "reading order"? 

I'm grateful for any tips!

**Comment by JorjMcKie** [comment, marked as answer]
Just for your question not feeling so lonely, here some non-answer:
I haven't analyzed LibreOffice's PDF outputs yet, but did some from MS Word's.
As is to be expected, there was (almost) nothing special. Text reading sequence can be expected to be normal. Of course you need to take care if your doc pages contain multi-column text, but that is an issue independent from Word / LibreOffice.

Some text glyphs did not deliver the expected characters though, but that may be a Word peculiarity.

**Reply by shueffner** [reply]
That's not a non-answer, no news is good news :-) 
Thank you, I appreciate it!
